defaults/main.yml

```diff
@@ -25,6 +25,7 @@ openio_oioswift_workers:
 openio_oioswift_max_clients: 1024
 
 openio_oioswift_log_level: INFO
+openio_oioswift_log_name: "OIO,{{ openio_oioswift_namespace }},oioswift,{{ openio_oioswift_serviceid }}"
```

Comment:
Please remove this from default.
The log name must be fixed.

Reply:
fixed